Issue#109

spider65 opened the pull request:
#109 modified button

spider65 committed:
modified GuiLabel for button

spider65 committed:
Add new GuiKey fot new button

spider65 committed:
new GPIO for button service

spider65 committed:
add key in section panel board for new service button

spider65 committed:
add GPIO setting for new service button

spider65 committed:
add Button service

alkcxy committed:
created enum GPIO to remove dependency from RPi.GPIO

alkcxy committed:
#109 use SwitchStatus for all boolean switch

alkcxy committed:
#109 removed unused assignments

alkcxy committed:
#109 removed type: ignore comments as useless, removed GPIO mock

spider65 committed:
modified button

alkcxy committed:
Merge branch 'master' into issue#109

spider65 committed:
eliminated label and modified colors button in enable/disable

spider65 committed:
delete obsolete Guilabel

spider65 committed:
modified tooltip

alkcxy commented on client.py:
```diff
@@ -5,7 +5,7 @@
 from gui_constants import GuiLabel, GuiKey
 from logger import LoggerClient
-from status import Status, TelescopeStatus, PanelStatus, TrackingStatus
+from status import Status, TelescopeStatus, SwitchStatus, TrackingStatus
```
I think maybe it's better to use ButtonStatus instead of SwitchStatus in accordance with ButtonControl

alkcxy commented on automazione_tende.py:
```diff
@@ -214,31 +217,61 @@ def close_roof(self) -> None:
 
-    def panel_on(self):
+    # PANEL FLAT
+    def on(self):
```
I'd keep panel_on and panel_off on this class

alkcxy commented on mock/telescope.ini:
```diff
@@ -0,0 +1,6 @@
+[coords]
```
please, remove this file from git

spider65 committed:
replace SwitchStatus with ButtonStatus
replace method on with panel_on
replace method off with panel_off
update getignore

spider65 commented:
replace SwitchStatus with ButtonStatus 
replace method on with panel_on replace method off with panel_off 
update getignore

alkcxy committed:
#109 removed telescope.ini

alkcxy committed:
#109 fixed failing tests, park telescope after unit test execution
